core/dbt/task/sql.py

```diff
@@ -28,18 +29,19 @@ def handle_exception(self, e, ctx):
                 exc=str(e), exc_info=traceback.format_exc(), node_info=self.node.node_info
             )
         )
+        # REVIEW: This code is invalid and will always throw.
```

Comment:
My guess here is that sometimes we were getting DbtRuntimeErrors without a node, and some of the message creation wouldn't work, but that subsequent refactoring has made it non-functional?  It seems like a pretty bad thing if it always throws. Does anything break if it's removed?